docs/02-app/01-building-your-application/02-data-fetching/01-fetching.mdx

```diff
@@ -97,7 +97,7 @@ let data = await fetch('https://api.vercel.app/blog', { cache: 'no-store' })
 
 ### Fetching data on the server with an ORM or database
 
-This component will always fetch and display a dynamic, up-to-date list of blog posts.
+This page will be SSG by default. The page will be prerendered during `next build`. Can be revalidated using [Incremental Static Regeneration](/docs/app/building-your-application/data-fetching/incremental-static-regeneration).
```

Comment:
```suggestion
This page will be SSG by default. The page will be prerendered during `next build`. It can be revalidated using [Incremental Static Regeneration](/docs/app/building-your-application/data-fetching/incremental-static-regeneration).
```